helix-core/src/test/java/org/apache/helix/integration/manager/ClusterControllerManager.java

```diff
@@ -37,6 +38,10 @@ public ClusterControllerManager(String zkAddr, String clusterName, String contro
     super(zkAddr, clusterName, controllerName, InstanceType.CONTROLLER);
   }
 
+  public ClusterControllerManager(String clusterName, HelixManagerProperty helixManagerProperty) {
```

Comment:
If this is for testing purpose, can we add @VisibleForTest or another similar decorator?

Reply:
We need this note even when the class is defined test module? I don't think this class has any non-test usage,

Reply:
Didn't notice, my bad.